pkg/cmd/server/origin/controller.go

```diff
@@ -0,0 +1,41 @@
+package origin
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	kapi "k8s.io/kubernetes/pkg/api"
+
+	"github.com/openshift/origin/pkg/cmd/server/origin/controller"
+)
+
+func (c *MasterConfig) NewOpenshiftControllerInitializers() (map[string]controller.InitFunc, error) {
+	ret := map[string]controller.InitFunc{}
+
+	// initialize build controller
+	storageVersion := c.Options.EtcdStorageConfig.OpenShiftStorageVersion
+	groupVersion := schema.GroupVersion{Group: "", Version: storageVersion}
+	codec := kapi.Codecs.LegacyCodec(groupVersion)
```

Comment:
looks like everyone wants a codec.  Add a TODO to add it to our controller context.